js/nina-common/src/contexts/release.js

```diff
@@ -867,21 +897,29 @@ const releaseContextHelper = ({
   }
 
   const getRelatedForRelease = async (releasePubkey) => {
-    const release = releaseState.tokenData[releasePubkey]
-    if (release) {
-      for await (let recipient of release.royaltyRecipients) {
-        if (recipient.percentShare.toNumber() > 0) {
-          await getReleasesHandler(
-            recipient.recipientAuthority,
-            lookupTypes.REVENUE_SHARE
-          )
-          await getReleasesHandler(
-            recipient.recipientAuthority,
-            lookupTypes.PUBLISHED_BY
-          )
-        }
+    let release = releaseState.tokenData[releasePubkey]
+    if (!release) {
+      release = await fetchRelease(releasePubkey)
+    }
+    const releaseIds = []
```

Comment:
should we add `release.royaltyRecipients.filter( recipient => recipient.id !== '...................')` to get the placeholder ones out before looping?